english/friendly-person/warg/warg.md

```diff
@@ -0,0 +1,3 @@
+
+You quickly Warg into the mind of a nearby Dire Wolf and creep toward your mysterious doppleganger but just before you get close enough to get their attention there is an explosion and you wake up lying in the snow, or is marshmellow?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.